server/package-lock.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Can this file be removed? There is already a `yarn.lock`, and I think it is better to stick with using yarn in this project than both yarn and npm

Reply:
Sure, I'm removing it. It was actually created/added by mistake.